documentation/modules/exploit/unix/ftp/proftpd_modcopy_exec.md

```diff
@@ -0,0 +1,94 @@
+## Vulnerable Application
+
+This module exploits the SITE CPFR/CPTO mod_copy commands in ProFTPD version 1.3.5.
+Any unauthenticated client can leverage these commands to copy files from any
+part of the filesystem to a chosen destination. The copy commands are executed with
+the rights of the ProFTPD service, which by default runs under the privileges of the
+'nobody' user. By using /proc/self/cmdline to copy a PHP payload to the website
+directory, PHP remote code execution is made possible.
+
+
+## Installation Steps
+
+Download and build:
+
+```sh
+wget ftp://ftp.proftpd.org/distrib/source/proftpd-1.3.5.tar.gz
+tar zxvf proftpd-1.3.5.tar.gz
+cd proftpd-1.3.5
+./configure --with-modules=mod_copy
+make
+```
+
+Run ProFTPD in foreground with `-n` flag for testing:
+
+```
+sudo ./proftpd -n
```

Comment:
Hey @bcoles I'm getting the follow error when attempting to install on Ubuntu 22.04.01. Did you run into this/ do you have a solution we might be able to add to the Installation Steps?
```
msfuser@msfuser-virtual-machine:~/proftpd-1.3.5$ sudo ./proftpd
2023-03-22 20:27:36,725 msfuser-virtual-machine proftpd[14779]: fatal: unable to read configuration file '/usr/local/etc/proftpd.conf': No such file or directory
```

Reply:
added